src/metabase/upload.clj

```diff
@@ -461,6 +464,10 @@
   [db schema-name]
   (nil? (can-create-upload-error db schema-name)))
 
+(defn- start-timer [] (System/nanoTime))
+
+(defn- check-timer-ms [timer] (/ (- (System/nanoTime) timer) 1e6))
```

Comment:
Hmmm.. how about `since-ms` as a name for this function? 😁 It's a nitpick, but sounds more pleasant. :)

Reply:
I'm still getting used to using nouns instead of verbs for clojure functions, but you're right that does read better ✨

Reply:
> I'm still getting used to using nouns instead of verbs for clojure functions

Native speaker problems, you know. To me it's all terminology, not words. 🤣